client/src/leap/soledad/client/sqlcipher.py

```diff
@@ -676,7 +676,7 @@ def close(self):
         Close the syncer and syncdb orderly
         """
         # close all open syncers
-        for url in self._syncers:
+        for url in self._syncers.keys():
```

Comment:
Is this really needed? An iteration over a `dict` will already return the keys in the `dict`.